Samples/BetaSamples/LocalMediaSamples/PolicyRecordingBot/WorkerRole/CRWorkerRole.csproj

```diff
@@ -82,17 +82,9 @@
     </Content>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-
-  <Target
-    Name="BuiltProjectOutputGroupDependencies"
-    DependsOnTargets="$(BuiltProjectOutputGroupDependenciesDependsOn)"
-    Returns="@(BuiltProjectOutputGroupDependency)">
+  <Target Name="BuiltProjectOutputGroupDependencies" DependsOnTargets="$(BuiltProjectOutputGroupDependenciesDependsOn)" Returns="@(BuiltProjectOutputGroupDependency)">
     <ItemGroup>
-      <BuiltProjectOutputGroupDependency Include="@(ReferenceCopyLocalPaths->'%(FullPath)');
```

Comment:
If there is no change here, please revert the change, prefer the earlier formatting